pom.xml

```diff
@@ -278,6 +278,12 @@
                         <enableProcessChecker>native</enableProcessChecker>
                         <trimStackTrace>false</trimStackTrace>
                         <redirectTestOutputToFile>true</redirectTestOutputToFile>
+                        <properties>
+                            <configurationParameters>
```

Comment:
`configurationParameters` does not exist: https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html

Either create a junit-platform.properties in every `src/test/resources` folder, OR use surefire's argLine.

The first approach will have the advantage that IntelliJ and others will pick it up. The second is DRY-compatible (but that may actually be a disadvantage)

Reply:
Tests run in parallel. Taken from here: https://rieckpil.de/parallelize-only-java-unit-tests-with-junit-5-and-maven/

Reply:
Found the docs:
https://maven.apache.org/surefire/maven-surefire-plugin/examples/junit-platform.html